onnxruntime/core/framework/allocation_planner.cc

```diff
@@ -2246,7 +2246,7 @@
   // For in-place reuse tensors, the lifetime is the union of all the tensors that tensors that use that buffer
 #if !defined(ORT_MINIMAL_BUILD) && defined(ORT_MEMORY_PROFILE)
   void AdjustInplaceLifeIntervals() {
-    InlinedHashMap<OrtValueIndex, std::vector<OrtValueIndex>> inplace_reuse_buffer;
+    InlinedHashMap<OrtValueIndex, InlinedVector<OrtValueIndex>> inplace_reuse_buffer;
     for (size_t i = 0; i < ort_value_info_.size(); ++i) {
       if (AllocPlan(OrtValueIndex(i)).inplace_reuse != OrtValueIndex(i)) {
         inplace_reuse_buffer[ort_value_info_[i].inplace_reused_buffer_index].push_back(OrtValueIndex(i));
```

Comment:
>inplace_reuse_buffer[ort_value_info_[i].inplace_reused_buffer_index]. [](http://example.com/codeflow?start=8&length=69)

use insert_or_assign instead of operator[]